tox.ini

```diff
@@ -149,15 +149,13 @@ commands =
 [testenv:flake8-tests]
 basepython = {[testenv:flake8]basepython}
 deps =
-    flake8
-    # https://github.com/JBKahn/flake8-print/pull/30
-    flake8-print>=3.1.0
+    -rdev_requirements/linter-requirements.txt
 commands =
     flake8 \
         # Ignore F811 redefinition errors in tests (breaks with pytest-mock use)
         # E203 is not PEP8 compliant https://github.com/ambv/black#slices
         # W503 is not PEP8 compliant https://github.com/ambv/black#line-breaks--binary-operators
-        --ignore F811,E203,W503 \
+        --ignore F811,E203,W503,D \
```

Comment:
What is the `D` for?

Reply:
Can we document the `D` in the comments above this line?

Reply:
```suggestion
        # D is for Documentation. We do not care if Test classes and their methods have docstrings
        --ignore F811,E203,W503,D \
```